buildroot-external/patches/lighttpd/0001-rega-auth/lighttpd/src/mod_authn_rega.c

```diff
@@ -93,7 +93,7 @@ int checkAuth(int port, const char* user, const char* pass) {
 
   int m = 0;
   //user
-  for(int i = 0 ; i < lengthUser && m <= 1023; i++) {
+  for(int i = 0 ; i < lengthUser && m <= 1022; i++) {
```

Comment:
@alexreinert korrigier mich wenn ich mich täusche - aber wenn man mehrere `\` oder `:` einfügt kann man doch immer noch ein overflow provozieren? Müsste man das hier nicht mit `m <= 511` begrenzen oder die länge nach dem escapen in der schleife prüfen?